site/docs/v1/tech/apis/scim/_scim-enterpriseuser-request-body.adoc

```diff
@@ -0,0 +1,47 @@
+include::/docs/v1/tech/apis/scim/_scim-common-user-header.adoc[]
+
+The following is an example of a typical SCIM User create request body a SCIM client might send. However, your incoming request lambda could use these values in completely different ways to populate the FusionAuth User object if you choose.
+
+This is taken from the SCIM Schema RFC describing a SCIM EnterpriseUser schema. For the full specification you can find the RFC link:https://datatracker.ietf.org/doc/html/rfc7643#section-4.3[here]
+
+==== Request Body
+[.api]
+include::/docs/v1/tech/apis/scim/_scim-common-body.adoc[]
+
+include::/docs/v1/tech/apis/scim/_scim-common-user-request-body.adoc[]
+
+[field]#urn:ietf:params:scim:schemas:extension:enterprise:2.0:User# [type]#[String]# [optional]#Required#::
+This is the root property of the EnterpriseUser schema extension. It should match the identifier included in the schemas property.
+
+[field]#urn:ietf:params:scim:schemas:extension:enterprise:2.0:User.employeeNumber# [type]#[String]# [optional]#Optional#::
+A string identifier, typically numeric or alphanumeric, assigned to a person, typically based on order of hire or association with an organization.
+
+[field]#urn:ietf:params:scim:schemas:extension:enterprise:2.0:User.costCenter# [type]#[String]# [optional]#Optional#::
+Identifies the name of a cost center.
+
+[field]#urn:ietf:params:scim:schemas:extension:enterprise:2.0:User.organization# [type]#[String]# [optional]#Optional#::
+Identifies the name of an organization.
+
+[field]#urn:ietf:params:scim:schemas:extension:enterprise:2.0:User.division# [type]#[String]# [optional]#Optional#::
+Identifies the name of a division.
+
+[field]#urn:ietf:params:scim:schemas:extension:enterprise:2.0:User.department# [type]#[String]# [optional]#Optional#::
+Identifies the name of a department.
+
+[field]#urn:ietf:params:scim:schemas:extension:enterprise:2.0:User.manager# [type]#[String]# [optional]#Optional#::
+The user's manager.  A complex type that optionally allows service providers to represent organizational hierarchy by referencing the "id" attribute of another User.
```

Comment:
FusionAuth is the service provider, but we don't have the concept of hierarchies like that so it would be up to the dev to figure out what they want to do with that data.